packages/core/package.json

```diff
@@ -5,8 +5,9 @@
   "main": "src/index.ts",
   "type": "module",
   "exports": {
-    ".": "./src/index.ts",
-    "./browser": "./src/browser.ts"
+    ".": "./dist/index.js",
+    "./constants": "./dist/constants.js",
+    "./schema": "./src/schema/types.ts"
```

Comment:
Hold your horses please. Let me check if I can avoid this Rollup 🙏 

Also I already have core setup with Rollup let's not do the same work twice